checks/utils/windowsversions.py

```diff
@@ -12,7 +12,7 @@
         (release: \s+ (?P<release>\d|\w+); \s+)?
         revision: \s+ (?P<revision>\d+);
         \s+
-        (?P<bits>\d+)-bit
+        (?P<arm>[ARM]{0,3})\s?(?P<bits>\d+)-bit
```

Comment:
Why is this checking for a character class of `[ARM]` repeated 0-3 times? Are there that many variations of how Windows on ARM can be represented?

Reply:
@wizardcm figured that out for me. I have no idea how regex works.

Reply:
It's checking for "ARM 64-bit".

Reply:
Use this instead:
```
(?P<arm>ARM)?\s?(?P<bits>\d+)-bit
```